### DISCUSSION TITLE
Idea Mapa del sitio

### DISCUSSION THREAD

**Discussion by ZafkielEdt** [FrancoGL/ISPC-FullStack#16] [Ideas Mapa del Sitio]
Esta es mi idea para el diseño del sitio. El mismo se compone de:
 * Header
 * Main
 * Section
 * Footer

[Diseño](https://www.figma.com/file/hFSHXMWtKJp5LYI1vUfkF7/Borrador-Destino-C%C3%B3rdoba?node-id=0%3A1)

**Comment by veroivk** [comment]
Finalmente se eligió trabajar sobre Figma. Muy buena tu idea Franco.

**Comment by federico42o** [comment]
con una imagen de fondo quedaría piola no? algo así [sugerencia](https://www.figma.com/file/kutsX2xRosh9cPcoEAt7EL/Untitled?node-id=0%3A1)
que dicen?

**Reply by ZafkielEdt** [reply]
Si, quedaría genial

**Reply by veroivk** [reply]
La verdad que sí. También podemos probar con alguna imagen del Dique San Roque o las sierras? No sé que opinan.

**Reply by ZafkielEdt** [reply]
> La verdad que sí. También podemos probar con alguna imagen del Dique San Roque o las sierras? No sé que opinan.

Habría que probar, lo ideal es que la imagen no interfiera con la visualización de los textos.

**Comment by veroivk** [comment]
Aquí les dejo mi aporte. Incorporé una imagen de los flamencos del Parque Nacional Ansenuza y nuevo subtitulo o epígrafe. Podría ser un hotel exclusivo. https://www.figma.com/file/vUnUH9xgyctuvm91MJLwO6/Flamencos-del-Parque-Nacional-Ansenuza?node-id=0%3A1

**Reply by ZafkielEdt** [reply]
Recuerda que la página no es de un hotel, sino que brinda info turística sobre Córdoba

**Reply by veroivk** [reply]
Pero no se podría incluir dentro del sitio? Porque tenemos que diferenciarnos del sitio de la secretaria de turismo.
